apidef/oas/default.go

```diff
@@ -77,6 +92,14 @@ func (s *OAS) BuildDefaultTykExtension(overRideValues TykExtensionConfigParams)
 		}
 	}
 
+	if overRideValues.AllowList != nil {
```

Comment:
can we create a function like `importMiddlewares`?

Reply:
rename this function?

Reply:
grouped both into a single function

Reply:
No, collecting middlewares inside a separate func